dev/doc/archive/Translator.tex

```diff
@@ -29,10 +29,10 @@
   {\begin{center}\tt\begin{tabular}{l|ll} \hfil\textrm{V7} & \hfil\textrm{V8} \\ \hline}
   {\end{tabular}\end{center}}
 \def\TRANS#1#2
-  {\begin{tabular}[t]{@{}l@{}}#1\end{tabular} & 
+  {\begin{tabular}[t]{@{}l@{}}#1\end{tabular} &
```

Comment:
Changes in 7 of these files only remove trailing spaces.  I though we tried to avoid doing this.  (Do whatever you think is right.)

Reply:
This is the pre-commit hook acting here. Since the file was moved, all the lines are considered to have changed.